### PR TITLE
Newsletter 12: Add Library & Tooling items

### PR DESCRIPTION
**Pull request by ozkriff**
@philip-lamb @zkat @Trisfald @naia-rs @phaazon, should something be changed or added? r? :)

_Part of #201 ("Newsletter 12: Coordination/Tracking")_

### CODE REVIEW

**Comment by ozkriff** [comment]
:timer_clock: Merging in a few hours if there're no objections.

File: content/posts/newsletter-012/index.md [hunk 1]
```diff
@@ -660,6 +660,90 @@ about their experience of using Rust in games at [Gameroasters].
 
 ## Library & Tooling Updates
 
+### [A Servo Browser Plugin for Unity][servo-unity-post]
```
Comment (kvark) [inline content/posts/newsletter-012/index.md:663]: lol, so finally Unity is going to support WebGPU by running it inside Serve browser embedded into a Unity scene 😆

**Comment by Trisfald** [comment]
perfect 👍

**Comment by ozkriff** [comment]
@phaazon You want me to pull your abovementioned commit here, right?

**Comment by philip-lamb** [comment]
LGTM. Thanks for the mention.

**Pull request merged**